fake.go

```diff
@@ -519,6 +520,80 @@ func (fake *Fake) Dump() string {
 	return buf.String()
 }
 
+var commentRegexp = regexp.MustCompile(`^(.*)comment "([^"]*)"( ;)?([^"]*)$`)
+
+// splitComment splits line into a required pre-comment and optional trailing comment
+// (which is enclosed in quotes but does not contain any quotes).
+func splitComment(line string) (string, *string) {
+	//fmt.Printf("Split comment !%v!\n", line)
+	// We could perhaps do this more efficiently without using a regexp, but it would
+	// be more complicated...
+	match := commentRegexp.FindStringSubmatch(line)
+	if match != nil {
+		return strings.TrimSpace(match[1]) + match[4], &match[2]
```

Comment:
Is the TrimSpace to get rid of the space before `comment`? You could put that into the regexp. ("` ?comment`")

Reply:
Yeah that won't work because the previous `(.*)` group is greedy, and will include space if it is present. I couldn't find a way to remove optional space from that group